apps/schema/static/schema/swagger.yaml

```diff
@@ -628,6 +653,14 @@ components:
         schema:
           $ref: '#/components/schemas/dataTypes/ethereum/transactionHash'
 
+      wallet_id:
+        required: true
+        name: wallet_id
+        in: query
+        description: Wallet ID to filter transactions by
+        schema:
```

Comment:
Something weird is going on with this query parameter in the documentation. It looks like it contains a child element, which isn't consistent with the way the rest of the query parameters look.
![image](https://user-images.githubusercontent.com/856738/49597603-c868af00-f94a-11e8-9c20-93b7b85538ff.png)